zproject/urls.py

```diff
@@ -167,6 +167,9 @@
     url(r'^config-error/github$', TemplateView.as_view(
         template_name='zerver/config_error.html',),
         {'github_error': True},),
+    url(r'^smtp-config-error$', TemplateView.as_view(
```

Comment:
Let's do `/config-error/smtp` for the URL.  I have an idea in mind of a refactor we could do to make these be just 1 `urls.py` entry, and that sorta depends on there being a structure to these...